persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/stringbased/JdbcStringBasedStore.java

```diff
@@ -322,6 +346,23 @@ public ConnectionFactory getConnectionFactory() {
       });
    }
 
+   private long runBatchAndNotify(List<PossibleExpirationNotification> possible, PreparedStatement batchDelete,
+         FlowableProcessor<MarshallableEntry<K, V>> flowable) throws SQLException {
+      long purgeAmount = 0;
+      int[] results = batchDelete.executeBatch();
+      for (int i = 0; i < results.length; ++i) {
+         if (results[i] != Statement.EXECUTE_FAILED) {
```

Comment:
Should we add a warning/debug log if `EXECUTE_FAILED`?

Reply:
I think that is probably a bit too verbose, probably just trace. This could be due to a concurrent write operation replacing/removing the entry so it is a possibly expected state.